UCSD/campus-mobile
# Busyness redesign

## Summary
This Pull Request implements the outlined redesign discussed in the following one-pager: https://ucsdcollab.atlassian.net/wiki/external/MTEwNDg3NTc4NDU5NDg3MDgzMjFjMTZiZDJjMTViMmY


## Changelog

[General] [Change] - redesigns business card for improved user experience


## Test Plan

Look for bugs in the following:
- Ensure individual busyness card pages are no longer vertically scrollable and now comfortably fit 3 locations per page
- Ensure busyness pages split across multiple pages (e.g., Dining Halls (1/2) and Dining Halls (2/2)) appear and are next to each other regardless of reordering (e.g., if you see a page EXAMPLE (1/3), that card should be immediately followed by pages of the form EXAMPLE (2/3) and EXAMPLE (3/3)).
- Try enabling, disabling, and reorganizaing locations via the "manage locations" button
- Perform any misc testing with the busyness card you can think of (e.g., view busyness card when logged in or logged out, try reordering and hiding busyness card, etc.)
